### PR TITLE
[TTAHUB-2617] Add recent columns to CSV export

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Add FEI goal id, CLASS goal id, root cause, goal antcipated date, next step dates to CSV file

## How to test
Confirm the following columns were added to the CSV export. Note some naming conventions do not 100% match the Jira spec. In those cases, I deferred to our existing implementation.

- AR language
- Goal "standard OHS goal". column contains "yes" or "no". 
- Goal root cause column. If the goal is not FEI, then it will be blank. Root causes will be distinctly aggregated. 
- Goal Source 
- Objective  support type
- Objective iPD course name, aggregate into one column if more then one iPD course name was selected
- After existing column "Specialist's next steps": "Specialist's next steps anticipated completion date"
- After existing column "Recipient's next steps": "Recipient's next steps anticipated completion date"

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2617

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -261,6 +261,8 @@ const GOAL_CREATED_VIA = ['imported', 'activityReport', 'rtr', 'merge', 'admin',
 
 const CURRENT_GOAL_SIMILARITY_VERSION = 3;
 
+const OHS_STANDARD_GOAL_TEMPLATE_IDS = [];
```
Comment (GarrettEHill) [inline src/constants.js:264]: Do we need to populate this array here or is it dynamic

File: src/lib/transform.js [hunk 2]
```diff
@@ -286,6 +355,11 @@ function makeGoalsAndObjectivesObject(objectiveRecords) {
       // Make sure its not another objective for the same goal.
       if (goalIds[goalName] && !goalIds[goalName].includes(goalId)) {
         accum[`goal-${existingObjectiveTitle}-id`] = `${accum[`goal-${existingObjectiveTitle}-id`]}\n${goalId}`;
+        accum[`goal-${goalNum}-source`] = uniq([...accum[`goal-${goalNum}-source`].split('\n'), goal.source]).join('\n');
+        if (goal.isCurated) {
+          accum[`goal-${goalNum}-fei-root-causes`] = `${accum[`goal-${goalNum}-fei-root-causes`]}\n${goal.responses
```
Comment (GarrettEHill) [inline src/lib/transform.js:360]: is there a reason this is not also running uniq?

File: src/constants.js (hunk 1 above)
Comment (GarrettEHill) [reply src/constants.js:264]: this does not appear to be used

File: src/lib/transform.js (hunk 2 above)
Comment (thewatermethod) [reply src/lib/transform.js]: good call

**Review by AdamAdHocTeam — COMMENTED** [review]
Getting an error exporting a report.

image

image

**Review by AdamAdHocTeam — COMMENTED** [review]
Also double check but I don't see the language column.

**Review by AdamAdHocTeam — APPROVED** [review]
Issues are fixed.

**Pull request merged**